Prevents:
error: cannot update the lock file /home/builder/.termux-build/librewolf/src/Cargo.lock
because --frozen was passed to prevent this

--- a/python/mozbuild/mozbuild/vendor/vendor_rust.py
+++ b/python/mozbuild/mozbuild/vendor/vendor_rust.py
@@ -735,7 +735,6 @@ license file's hash.
         flags = ["--output-format=json"]
         if "MOZ_AUTOMATION" in os.environ:
             flags.append("--locked")
-            flags.append("--frozen")
         res = cargo_vet(
             self,
             flags,
--- a/build/moz.configure/rust.configure
+++ b/build/moz.configure/rust.configure
@@ -1061,7 +1061,7 @@ set_config("MOZ_CARGO_DEFAULT_PROFILE_ARGS", rust_cargo_profile.default_args)
 # tree, so we want to let Cargo create lock files in this case. When built
 # within a tree, the Rust dependencies have been vendored in so Cargo won't
 # touch the lock file.
-set_config("MOZ_CARGO_FROZEN_ARGS", ["--frozen"], when=~js_standalone)
+set_config("MOZ_CARGO_FROZEN_ARGS", [], when=~js_standalone)
 
 # With TSan, we also need to rebuild the rust stdlib so that it's instrumented.
 # Because build-std is still pretty experimental, we need to explicitly request
--- a/build/RunCbindgen.py
+++ b/build/RunCbindgen.py
@@ -55,7 +55,7 @@ def generate_metadata(output, cargo_config):
     # within a tree, the Rust dependencies have been vendored in so Cargo won't
     # touch the lock file.
     if not buildconfig.substs.get("JS_STANDALONE"):
-        args.append("--frozen")
+        pass
 
     stdout, returncode = _run_process(args)
 
